src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -594,5 +594,81 @@ module.exports.external = [
         func: function (o) {
             return isNaN(o);
         }
+    },
+    {
+        name: 'abs',
+        description: 'Returns the absolute value of a number.',
+        func: function (x) {
+            return Math.abs(x);
+        }
+    },
+    {
+        name: 'ceil',
+        description: 'Returns the next largest whole number or integer.',
+        func: function (x) {
+            return Math.ceil(x);
+        }
+    },
+    {
+        name: 'floor',
+        description: 'Returns the largest integer less than or equal to a given number.',
+        func: function (x) {
+            return Math.floor(x);
+        }
+    },
+    {
+        name: 'max',
+        description: 'Returns the largest of zero or more numbers.',
+        func: function () {
+            var args = [];
+            for (var i = 0; i < arguments.length - 1; i++)
+                args[i] = arguments[i];
+            return Math.max(...args);
+        }
+    },
+    {
+        name: 'min',
+        description: 'Returns the lowest-valued number passed into it, or NaN if any parameter isn\'t a number and can\'t be converted into one.',
+        func: function () {
+            var args = [];
+            for (var i = 0; i < arguments.length - 1; i++)
+                args[i] = arguments[i];
+            return Math.min(...args);
```

Comment:
nit : will arguments.slice(0, -1) work here?

Reply:
Unfortunately, arguments object is not an array, it is object which behaves somewhat array like. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments 
So, it does not have a slice method.